node/nodeRunner.go

```diff
@@ -384,6 +384,14 @@ func (nr *nodeRunner) executeOneComponentCreationCycle(
 		return true, err
 	}
 
+	selfId := managedBootstrapComponents.ShardCoordinator().SelfId()
```

Comment:
please re-check this. On rc/v1.4.0 we do not have this call here

Reply:
in rc/v1.4.0 these are called in the `addSyncersToAccountsDB` function after setting the trie syncers.